mix.exs

```diff
@@ -2,16 +2,16 @@ defmodule MMDB2Decoder.MixProject do
   use Mix.Project
 
   @url_github "https://github.com/elixir-geolix/mmdb2_decoder"
+  @version "3.1.0-dev"
 
   def project do
     [
       app: :mmdb2_decoder,
       name: "MMDB2 Decoder",
-      version: "3.1.0-dev",
+      version: @version,
       elixir: "~> 1.7",
       aliases: aliases(),
       deps: deps(),
-      description: "MMDB2 File Format Decoder",
```

Comment:
The description should imho stay here because this will be returned from `Application.started_applications/0`.